project/SparkBuild.scala

```diff
@@ -269,7 +269,8 @@ object SparkBuild extends PomBuild {
         // SPARK-45627 `enum`, `export` and `given` will become keywords in Scala 3,
         // so they are prohibited from being used as variable names in Scala 2.13 to
         // reduce the cost of migration in subsequent versions.
-        "-Wconf:cat=deprecation&msg=it will become a keyword in Scala 3:e"
+        "-Wconf:cat=deprecation&msg=it will become a keyword in Scala 3:e",
+        "-Wconf:cat=deprecation&msg=Unicode escapes in triple quoted strings are deprecated:e"
```

Comment:
this will be compile error in Scala 3?

Reply:
Only for test in GA.
I want it to run through GA first and see if there are any other similar issues besides the above modifications.
I will delete it later.

Reply:
this will nothing in Scala 3.3.1, neither a warning nor an error.
- scala 3.3.1
   <img width="443" alt="image" src="https://github.com/apache/spark/assets/15246973/d843e23e-6e06-4121-9e91-19807bf7bc1f">

- scala 2.13.8
   <img width="774" alt="image" src="https://github.com/apache/spark/assets/15246973/286fc68a-84c1-48cf-9716-7edafa2faa06">